src/content/docs/query-your-data/explore-query-data/query-builder/introduction-new-data-explorer.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Introduction to the data explorer
+tags:
+  - Query your data
+  - Explore and query data
+  - Data explorer
+metaDescription: 'The New Relic data explorer lets you run queries of your data without using NRQL-format queries'
+redirects:
+  - /docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#query-from-anywhere
+
+freshnessValidatedDate: never
+---
+
+Navigate all your data visually using the <DNT>**query experience data explorer**</DNT> and without needing to know how to write [NRQL-format queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
+
+Need to perform a more detailed search? Read how to [query your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/). If you haven't already, create your free New Relic account to start monitoring your data today.
+
+
+## Why It Matters [#why-it-matters]
```

Comment:
```suggestion
## Understand query your data [#understand-query-buildler]
```

Reply:
I think "why it matters" can be something people read by really quickly because it's more conceptual. this is a suggestion, but it's not a blocker :)